sshd-core/src/main/java/org/apache/sshd/client/channel/PtyCapableChannelSession.java

```diff
@@ -200,21 +199,6 @@ public void setPtyModes(Map<PtyMode, Integer> ptyModes) {
         config.setPtyModes((ptyModes == null) ? Collections.emptyMap() : ptyModes);
     }
 
-    /**
-     * @param  key   The (never {@code null}) key (Note: may be empty...)
-     * @param  value The value to set - if {@code null} then the pre-existing value for the key (if any) is
-     *               <U>removed</U>.
-     * @return       The replaced/removed previous value - {@code null} if no previous value set for the key.
-     */
-    public Object setEnv(String key, Object value) {
-        ValidateUtils.checkNotNull(key, "No key provided");
```

Comment:
Please use `ValidateUtils#checkNotNullAndNotEmpty(key)`

Reply:
it is just deleted code :)